docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,159 @@
+JDBC Transactional POJO Output Operator
+==============
+
+## Operator Objective
+This operator receives an input stream of POJOs and inserts them as rows in a database table in a fault-tolerant way.
+
+## Overview
+The main features of this operator are persisting data to the database table and fault tolerance. This operator creates a transaction at the start of window, executes batches of sql updates, and closes the transaction at the end of the window. Each tuple corresponds to an SQL update statement. The operator groups the updates in a batch and submits them with one call to the database. Batch processing improves performance considerably. The size of a batch is configured by batchSize property. The tuples in a window are stored in check-pointed collection which is cleared in the endWindow(). This is needed for the recovery. The operator writes a tuple exactly once in the database, which is why only when all the updates are executed, the transaction is committed in the end window call. 
+This operator internally extends the functionality of AbstractPassThruTransactionableStoreOutputOperator.This is the base implementation for a pass-through output adapter of a transactional store.This implementation provides the "exactly once" guarantee."Pass-through" means it does not wait for end window to write to the store. It will begin transaction at begin window and write to the store as the tuples come and commit the transaction at end window.
```

Comment:
AbstractPassThruTransactionableStoreOutputOperator.This => `AbstractPassThruTransactionableStoreOutputOperator`. This
store.This => store. This
guarantee."Pass-through" => guarantee. "Pass-through"
begin transaction at begin window => begin a transaction at `beginWindow`
transaction at end window => transaction at `endWindow`

Not clear if "This implementation" refers to the current class or the base class; suggest rewriting so that it is clear which class is meant by the various occurrences of "it".